### PR TITLE
feat: frontend part of bitcoin tax import support

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/locales/cn.json [hunk 1]
```diff
@@ -2452,37 +2452,37 @@
...
+      "import": "从 Bisq 导入 {0}",
       "import_trade": "交易",
       "line_one": "[Experimental] 如果您发现导入的数据有问题，请告诉我们，以便我们相应地调整代码。",
-      "name": "Bisq",
-      "note": "Important notes for importing data from {0}'s CSV exports.",
-      "source_name": "Bisq"
+      "name": "Bisq"
+    },
+    "bitcoin_tax": {
+      "import": "从 BitcoinTaxes 导入 {0}",
```
Comment (kelsos) [inline frontend/app/src/locales/cn.json:2464]: Shouldn't this be BitcoinTax or something?

File: frontend/app/src/locales/en.json [hunk 2]
```diff
@@ -2536,37 +2536,37 @@
...
+      "import": "Import {0} from Bisq",
       "import_trade": "trades",
       "line_one": "[Experimental] If you find an issue with the imported data, let us know so we can adapt the code accordingly.",
-      "name": "Bisq",
-      "note": "Important notes for importing data from {0}'s CSV exports.",
-      "source_name": "Bisq"
+      "name": "Bisq"
+    },
+    "bitcoin_tax": {
+      "import": "Import {0} from BitcoinTaxes",
```
Comment (kelsos) [inline frontend/app/src/locales/en.json:2548]: @dimyG is this text correct here? is this how it should appear?

Comment (LefterisJP) [reply frontend/app/src/locales/en.json:2548]: It should be [bitcoin.tax](https://bitcoin.tax/)

**Review by kelsos — APPROVED** [review]
lgtm @lukicenturi thank you

**Pull request merged**